jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java

```diff
@@ -352,207 +298,204 @@ public void testIsWarProject() {
-        Arrays.asList(
+    assertThat(layers.dependenciesLayer.getEntries())
+        .comparingElementsUsing(SOURCE_FILE_OF)
+        .containsExactly(
             testRepository.artifactPathOnDisk("com.test", "dependency", "1.0.0"),
             dependenciesPath.resolve("more/dependency-1.0.0.jar"),
             dependenciesPath.resolve("another/one/dependency-1.0.0.jar"),
             dependenciesPath.resolve("libraryA.jar"),
             dependenciesPath.resolve("libraryB.jar"),
-            dependenciesPath.resolve("library.jarC.jar")),
-        layers.dependenciesLayers.get(0).getEntries());
-    assertSourcePathsUnordered(
-        Arrays.asList(
-            testRepository.artifactPathOnDisk("com.test", "dependencyX", "1.0.0-SNAPSHOT")),
-        layers.snapshotsLayers.get(0).getEntries());
-    assertSourcePathsUnordered(Arrays.asList(jar), layers.extraFilesLayers.get(0).getEntries());
-
-    assertExtractionPathsUnordered(
-        Arrays.asList(
-            "/app-root/libs/dependency-1.0.0-200.jar",
-            "/app-root/libs/dependency-1.0.0-480.jar",
-            "/app-root/libs/dependency-1.0.0-770.jar",
-            "/app-root/libs/library.jarC.jar",
-            "/app-root/libs/libraryA.jar",
-            "/app-root/libs/libraryB.jar"),
-        layers.dependenciesLayers.get(0).getEntries());
-    assertExtractionPathsUnordered(
-        Arrays.asList("/app-root/libs/dependencyX-1.0.0-SNAPSHOT.jar"),
-        layers.snapshotsLayers.get(0).getEntries());
-    assertExtractionPathsUnordered(
-        Arrays.asList("/app-root/classpath/final-name.jar"),
-        layers.extraFilesLayers.get(0).getEntries());
+            dependenciesPath.resolve("library.jarC.jar"));
+    assertThat(layers.snapshotsLayer.getEntries())
+        .comparingElementsUsing(SOURCE_FILE_OF)
+        .containsExactly(
+            testRepository.artifactPathOnDisk("com.test", "dependencyX", "1.0.0-SNAPSHOT"));
+    assertThat(layers.extraFilesLayer.getEntries())
+        .comparingElementsUsing(SOURCE_FILE_OF)
+        .containsExactly(jar);
+
+    assertThat(layers.dependenciesLayer.getEntries())
+        .comparingElementsUsing(EXTRACTION_PATH_OF)
+        .containsExactly(
+            "/my/app/libs/dependency-1.0.0-200.jar",
+            "/my/app/libs/dependency-1.0.0-480.jar",
+            "/my/app/libs/dependency-1.0.0-770.jar",
+            "/my/app/libs/library.jarC.jar",
+            "/my/app/libs/libraryA.jar",
+            "/my/app/libs/libraryB.jar");
+    assertThat(layers.snapshotsLayer.getEntries())
+        .comparingElementsUsing(EXTRACTION_PATH_OF)
+        .containsExactly("/my/app/libs/dependencyX-1.0.0-SNAPSHOT.jar");
+    assertThat(layers.extraFilesLayer.getEntries())
+        .comparingElementsUsing(EXTRACTION_PATH_OF)
+        .containsExactly("/my/app/classpath/final-name.jar");
   }
 
   @Test
-  public void testCreateContainerBuilder_warNonDefaultAppRoot()
+  public void testCreateContainerBuilder_war()
```

Comment:
This can be split into two tests? One to check for the source files and another to check the extraction paths- similar to what's being done in GradleProjectProperties?

Reply:
Done.